api/src/pcapi/templates/admin/support_beneficiary_details.html

```diff
@@ -250,6 +250,42 @@ <h3>Revue manuelle</h3>
   </div>
   {% endif %}
 </div>
+
+<h3>Historique des changements d'adresse email</h3>
+<div class="row">
+  {% if model.email_history|length > 0 %}
+  <div class="col-lg-12">
+    <a href="/pc/back-office/user_email_history/?search=pctest&flt1_0={{ model.id }}">Vue détaillée</a>
+    <table class="table table-striped table-hover">
+      <thead>
+        <tr>
+          <th scope="col">Ancienne adresse email</th>
+          <th scope="col">Nouvelle adresse email</th>
+          <th scope="col">Date</th>
+          <th scope="col">Type d'évenement</th>
+          <th scope="col">Device ID</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for email_history in model.email_history %}
+        <tr>
+          <td>{{ email_history.oldEmail }}</td>
+          <td>{{ email_history.newEmail }}</td>
+          <td>{{ email_history.creationDate.strftime('le %d/%m/%Y à %H:%M:%S') }}</td>
```

Comment:
(détail) l'heure est stockée en UTC, il faudrait le préciser ici :

```suggestion
          <td>{{ email_history.creationDate.strftime('le %d/%m/%Y à %H:%M:%S' UTC) }}</td>
```

Reply:
j'ai l'impression qu'on ne le précise pas ailleurs

est-ce que ça ne risque pas d'apporter de la confusion ?

Reply:
ok pour l'omettre, comme tu le sens. (Dans un monde idéal on stockerait des datetimes avec une timezone et on les afficherait dans la timezone locale de l'utilisateur. Ça viendra, un jour !)